api/services/report/javarosa-parser.js

```diff
@@ -0,0 +1,58 @@
+const _ = require('underscore'),
```

Comment:
Hopefully these parsers have just been moved from a different location...

Reply:
Yeah they were all in /packages/